ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -200,10 +211,62 @@ export const DataUseAgreementContent = () => {
 
     <SecondHeader>Data Disclaimer:</SecondHeader>
     <p>The <AoUTitle/> does not guarantee the accuracy of the data in
-      the <AoUTitle/> database. The <AoUTitle/>
-      does not guarantee the performance of the software in the <AoUTitle/>
-      database. The <AoUTitle/> does not warrant or endorse
+      the <AoUTitle/> database. The <AoUTitle/> does
+      not guarantee the performance of the software in the <AoUTitle/> database.
+      The <AoUTitle/> does not warrant or endorse
       the research results obtained by using the <i>All of Us</i> database.
     </p>
+
+    <SecondHeader>Terms and Definitions:</SecondHeader>
+    <IndentedUnorderedList>
+      <IndentedListItem>The <a href={{AOU_DEFINITION_URL}} target='_blank'><AoUTitle/></a> is a national longitudinal
```

Comment:
this links to `http://localhost:4200/[object%20Object]` for me.

Reply:
Already fixed! :D